src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/DisasterRecoveryService.cs

```diff
@@ -372,8 +387,9 @@ private bool IsBackupRestoreOperationSupported(string ownerUri, out ConnectionIn
                     }
                 }
             }
-            catch
```

Comment:
Usually these kinds of catches are used to catch exceptions from unmanaged code, which is something to keep in mind if that is still a potential risk.

Reply:
In this case we should just be using a `using` anyways to ensure the connection is closed even if an error happens

Reply:
All exceptions are derivatives of Exception class, so I don't expect any change in behavior.

Reply:
Was curious, so looked it up and turns out that is not necessarily true - although you have to specifically disable wrapped exceptions.

https://learn.microsoft.com/en-us/dotnet/csharp/how-to/how-to-catch-a-non-cls-exception

That being said - this shouldn't be an issue in STS.

Reply:
It seems to cause test failure, I'll revert this part as it's not critical.